data/ext/pending/issue-2636-examples.txt

```diff
@@ -0,0 +1,63 @@
+TYPES: Quiz, Question, eduQuestionType, Answer, hasPart, assesses, about, educationalAlignment
+
+PRE-MARKUP:
+
+Quiz is a test of knowledge, skills, or abilities; it can contain one-to-multiple questions with answers.
+
+MICRODATA:
+
+<!-- JSON-LD example only -->
+
+RDFA:
+
+<!-- JSON-LD example only -->
+
+JSON:
+
+
+<script type="application/ld+json">
+{
+  "@context": "https://schema.org/",
+  "@type": "Quiz",
+  "name": "Quiz about fractions",
+  "hasPart": {
+    "@type": "Question",
+    "url": "https://www.test.example.com/practice-problems-fractions",
+    "educationalAlignment": [
+      {
+        "@type": "AlignmentObject",
+        "alignmentType": "educationalSubject",
+        "targetName": "Mathematics"
+      },
+      {
+        "@type": "AlignmentObject",
+        "alignmentType": "educationalSubject",
+        "targetName": "Algebra"
+      },
+      {
+        "@type": "AlignmentObject",
+        "alignmentType": "educationalLevel",
+        "targetName": "Grade 5 » Number & Operations—Fractions » Apply and extend previous understandings of multiplication and division. » 3",
+        "educationalFramework": "Common Core",
+        "targetUrl": "http://www.corestandards.org/Math/Content/5/NF/B/3/"
+      }
+    ],
+    "about": "Fractions",
+    "educationalLevel": "intermediate",
+    "eduQuestionType": "Multiple choice",
+    "learningResourceType": "Practice problem",
+    "assesses": "Multiplying fractions by whole numbers",
+    "name": "Choose the model",
+    "comment": {
+      "@type": "Comment",
+      "text": "This is a hint about the question."
+    },
+    "encodingFormat": "text/markdown",
+    "text": "Which model represents the product $$2 * \frac{2}{3}$$?",
+    "suggestedAnswer": [
+      {
+        "@type": "Answer",
+        "@id": "http://example.org/fractions/practice-problem#answer_2",
+        "encodingFormat": "text/html",
+        "text": "<img src=\"www.test.com/practice-problems-fractions/…image2.png\" alt=\"Answer 2\" />"
```

Comment:
What is this terrifying thing? :) 

HTML inside a JSON(-LD) "text" field, itself likely already inside HTML via <script>. If the goal is to work out how to put small pieces of HTML inside Schema.org graphs, that seems a reasonable goal, but I don't think this combination of /encodingFormat and /text really work. The /text field isn't a good place for complex computer formats. Sorry if I missed this when we talked through this proposal recently.

Reply:
There are various ways we could put chunks of HTML inside the schema.org graph datamodel:

 * items in the graph could have an /asHTML property pointing to an instance of the existing /WebPageElement type, and then the actual markup could be a /rawHTMLText property from that (or similar names)
 * alternatively, just have /rawHTMLText on items in the graph, like the /Answer (this would be simpler than trying to pair /encodingFormat and /text)

/cc @vholland @rvguha

Reply:
Do you need the HTML inside the graph when the JSON-LD is inside the HTML? You have a fragment id there, why not put the answer in the HTML wrapped in `<div id="answer_2"></div>` That's what we did for [K12-OCX](https://k12ocx.github.io/k12ocx-specs/inpage/structure/) (there are analagous examples at the end)

Reply:
Ok, talking to @lucy-kind and colleagues, we'll retract this bit of the example for now and come up with another idiom